docker-compose.yml

```diff
@@ -2,8 +2,18 @@ version: '3'
 
 services:
   algod:
+    image: wwinder/algod:latest
```

Comment:
Hey, I was reading this PR. Not knowing much about docker config, but it seems we are going to replace docker image from `makerxau/algorand-sandbox-dev` to `wwinder/algod`?

It seems to me that, both are working under `DEV_MODE`, seemingly not too much perf change. Is it because Will is maintaining these image regularly?

Reply:
If we're going to write true multi-round e2e tests in addition to dry run tests, then yes, a `DEV_MODE` version would be useful. 

@algojohnlee @winder - is having a dev mode docker image variant on the roadmap as well?

Reply:
It is supported, set `DEV_MODE: true` to enable devmode, see the environment variables section: https://hub.docker.com/r/wwinder/algod

I believe the "latest" tag is stale because I don't delete it before adding the new tags, I should be able to fix that.

Reply:
I see that you're already using dev mode, does it not work the way you expect it?

Reply:
Thanks for clarifying!!! It's working well, but the _theoretical_ issue remains that fingers are crossed that the image will be ready in time for when it's used in tests.

Reply:
> I believe the "latest" tag is stale because I don't delete it before adding the new tags

Just one question, if we want to specify the latest version of your image, what tag should we specify? Or we have to pin a SHA?

Reply:
You're specifying the correct tag, I just haven't implemented it properly